### PR TITLE
Update free-courses-hi.md - add ethical hacking course hindi

### PR DESCRIPTION
**Pull request by krithikha24**
## What does this PR do?
Add resource(s) | This PR adds resources for Ethical Hacking courses in Hindi.

Remove resource(s) | Add info | Improve repo

### Description
Ethical hacking is a growing field and very useful. The course provided starts from scratch and also provides a project experience.

### Why is this valuable (or not)?
It is fully available on youtube for free.

### For book lists, is it a book? For course lists, is it a course? etc.
It is a course

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-hi.md
Comment (sohansai) [inline courses/free-courses-hi.md:675]:
```suggestion
* [Complete Ethical Hacking Tutorial for Beginners to Pro 2022](https://www.youtube.com/playlist?list=PLjVLYmrlmjGea8U9nphmCHGK_v6p_wq-R) - WsCube Tech
```

Comment (sohansai) [reply courses/free-courses-hi.md:675]: use the original title from the resource and avoid inventing new ones.

**Comment by krithikha24** [comment]
Commited the changes, Please approve now!

**Comment by krithikha24** [comment]
Linter error is fixed. Please approve and merge!

File: courses/free-courses-hi.md
Comment (rng70) [inline courses/free-courses-hi.md:686]: move line 689 to here

File: courses/free-courses-hi.md
Comment (rng70) [inline courses/free-courses-hi.md:689]: move line 689 to 686. it will resolve the linter issue

**Review by rng70 — CHANGES_REQUESTED** [review]

**Comment by krithikha24** [comment]
Resolved the said changes and waiting for approval

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**
